CONTRIBUTING.md

```diff
@@ -71,6 +71,32 @@ As a quick reference, the format of each page should match the following templat
 For more detailed page formatting guidelines,
 refer to the [style guide](contributing-guides/style-guide.md).
 
+## Translations
+
+Translation of pages can be done by simply creating the corresponding page in the appropriate top-level language specific directory under the appropriate platform subdirectory.
+When translating a page into a language that is not present yet, a new language specific directory must be created first.
```

Comment:
I'd suggest simplifying here too, merging this with the next paragraph:

> If the language-specific directory for the desired language does not yet exist, it must be created first, following the pattern ...

Reply:
What about:

> Translation of pages can be done by simply creating the corresponding page within the appropriate language-specific directory, creating that as well if it does not already exist.